nengo_gui/static/datastore.js

```diff
@@ -146,3 +139,177 @@ Nengo.DataStore.prototype.get_last_data = function() {
     }
     return shown;
 }
+
+
+Nengo.VariableDataStore = function(dims, sim, synapse){
```

Comment:
Would be `ResizeableDataStore` be a better name?
`VariableDataStore` might make one think of a `DataStore` for variables.